doc/source/user/how-to-how-to.rst

```diff
@@ -107,7 +107,7 @@ part of NumPy (like its API) but aren't obligated to paint a broader picture.
 
 For more on tutorials, see the `tutorial how-to`_.
 
-.. _`tutorial how-to`: https://github.com/numpy/numpy-tutorials/blob/master/tutorial_style.ipynb
+.. _`tutorial how-to`: https://github.com/numpy/numpy-tutorials/blob/main/content/tutorial-style-guide.md
```

Comment:
I'm actually not certain of the correct link here. When was this content available? Was it `.ipynb` or `.md`? The rendered version is not up (yet?) There seems to be some recent discussion here: https://github.com/numpy/numpy-tutorials/pull/81/files#r645955648